src/services/BookingService.ts

```diff
@@ -0,0 +1,97 @@
+import { Brackets } from "typeorm";
+import bookingRepository from "../repositories/BookingRepository";
+import IRepository from "../types/IRepository";
+import Booking from "../entities/Booking";
+import BookingCreateBodyValidator from "../validators/Booking/BookingCreateBodyValidator";
+import SlotService from "./SlotService";
+import HttpException from "../exceptions/HttpException";
+
+
+class BookingService {
+
+  public getRepository(): IRepository<Booking> {
+    return bookingRepository();
+  }
+
+  public async isSlotAvalible(
```

Comment:
Let's try to avoid other entities being the main subject of our questions in a service. If you name a method `isSlotAvailable` then it sounds like it should belong to Slot entity, but I agree that we are handling business logic about booking here so it makes sense to have this method implementation here.

I'd suggest renaming this method to something like `isBookingAvailable`.